docs/examples/video/video_label_example.py

```diff
@@ -56,14 +56,15 @@ def define_graph(self):
     for i in range(ITER):
         print("Iteration " + str(i))
         sequences_out, label = pipe.run()
-        sequences_out = sequences_out.as_cpu().as_array()
+        sequences_out = sequences_out.as_cpu()
```

Comment:
I would recommend to add test in python or in dali/pipeline/operators/reader/video_reader_op_test.cc that video with a different resolutions. For that you would need to add appropriate files to https://github.com/NVIDIA/DALI_extra/ repo as well and enable/disable this test depending on the driver version.

Reply:
Added PR in dali_extra
https://github.com/NVIDIA/DALI_extra/pull/20

Reply:
Merged